src/dashboard.js

```diff
@@ -773,13 +773,28 @@ function renderHistogramEvolution(lines, minDate, maxDate) {
   var drawnLines = lines.filter(function(line) { return !line.disabled; });
   
   // Filter out the points that are outside of the time range
+  var minX = maxDate, maxX = minDate;
   var filteredDatasets = drawnLines.map(function (line) {
     return {
       label: line.key,
       strokeColor: line.color,
-      data: line.values.filter(function(point) { return point.x >= minDate && point.x <= maxDate; }),
+      data: line.values.filter(function(point) {
+        return point.x >= minDate && point.x <= maxDate;
+      }).map(function(point) {
+        if (point.x < minX) { minX = point.x; }
+        if (point.x > maxX) { maxX = point.x; }
+        return point;
+      }),
     };
-  })
+  });
+  
+  // Add a fake series to expand the bounds a bit, which makes the chart look nicer when the timescale is small (within a day or so)
+  filteredDatasets.push({
+      data: [{x: minX - 1000, y: 2798}, {x: maxX + 1000, y: 2798}],
```

Comment:
Oops, should've tested with some smaller values as well - should be fixed as of ac7c5632d12bbb8af7c98d966738e13562a58f08.